sdv/metadata/multi_table.py

```diff
@@ -13,6 +15,165 @@ def __init__(self):
         self._tables = {}
         self._relationships = []
 
+    @staticmethod
+    def _validate_missing_relationship_keys(parent_table, parent_table_name, parent_primary_key,
+                                            child_table_name, child_foreign_key):
+        missing_keys = set()
+        parent_primary_key = cast_to_iterable(parent_primary_key)
```

Comment:
I think there are other places in this code that could now use the `cast_to_iterable` method. There are for sure some in the `SingleTableMetadata`